src/store/CurrentAccount/state.ts

```diff
@@ -0,0 +1,8 @@
+import { CurrentAccountInterface, StateIF } from '@/interfaces'
+
+export const state: CurrentAccountInterface = {
```

Comment:
We've been naming interfaces , "IF". Are you advocating to change that?

Reply:
Nope, I didn't intend to change that.  I'll correct it.